plugins/woocommerce-admin/client/core-profiler/style.scss

```diff
@@ -122,3 +124,123 @@
 		}
 	}
 }
+.woocommerce-profiler-business-location {
+	display: flex;
+	flex-direction: column;
+	.woocommerce-profiler-business-location__content {
+		max-width: 550px;
+		width: 100%;
+		display: flex;
+		align-self: center;
+
+		& > div {
+			width: 100%;
+		}
+
+		.woocommerce-profiler-heading {
+			margin-top: 72px;
+			@include breakpoint( '<782px' ) {
+				margin-top: 52px;
+				margin-bottom: 40px;
+				h1,
```

Comment:
I think we need to set h1 font-size to `32px` for mobile devices.


![Screenshot 2023-05-10 at 11 33 08](https://github.com/woocommerce/woocommerce/assets/4344253/2180fda5-9b26-4d3b-8390-8598c50a7afc)

And set h2's margin to 0.


![Screenshot 2023-05-10 at 11 34 29](https://github.com/woocommerce/woocommerce/assets/4344253/a3cab0cc-0670-4763-b75f-097c1b7f5b7c)


![Screenshot 2023-05-10 at 11 35 08](https://github.com/woocommerce/woocommerce/assets/4344253/fd1ba718-6282-47f1-8ae5-ac2b5b1e21cc)